MusicTaste/MusicTaste/View Controllers/Login/LoginViewController.m

```diff
@@ -106,5 +102,22 @@ - (void)presentAlertControllerWithTitle:(NSString *)title
                          completion:nil];
     });
 }
+- (void)methodAWithCompletion:(void (^) (BOOL success))completion
```

Comment:
The name `methodAWithCompletion` is not very descriptive; can you think of a better name for this?